crates/telemetry-subscribers/src/lib.rs

```diff
@@ -252,7 +255,7 @@ impl TelemetryConfig {
             panic_hook: true,
             crash_on_panic: false,
             prom_registry: None,
-            sample_nth: None,
+            sample_rate: 1.0,
```

Comment:
does this set the sample rate to 100% by default?

Reply:
yes - we can change this if for some reason its problematic to set the sample rate explicitly in prod